docs/docs/conceptual/gatsby-for-ecommerce.md

```diff
@@ -38,10 +40,17 @@ E-commerce tends to have a number of specific requirements. When building a Gats
 - **Handling checkout.** In order to comply with PCI regulations around storing credit card information, e-commerce providers typically exert strong control over the "buy" or "checkout" experience. Shopify requires all checkout flows using their platform to use their hosted checkout pages, though it's common to wrap them in a subdomain of the main site (e.g. the Gatsby/Shopify site [strivectin.com](strivectin.com] wraps a `myshopify.com` URL under a `shop.strivectin.com` URL for checkout).
 - **Handling account pages.** Again, many sites choose to wrap their e-commerce provider’s account pages under their own domain.
 
-Additional resources:
+## Additional resources:
+
+<Announcement style={{ marginBottom: "1.5rem" }}>
```

Comment:
I'm unsure whether or not we want to use this component for situations like this, and can remove this if not.